adr/0006-supported-node-and-npm-versions.md

```diff
@@ -0,0 +1,41 @@
+# Supported Node Versions
+
+Date: 2017-12-15
+
+## Status
+
+Proposed
+
+## Context 
+
+eLife has projects built in Node.
+ 
+eLife has projects that use the Node Package Manager (npm) in at least part of their own build process, even if the main technology of the project is not Node.
+
+In order to provide language version consistency across projects we need to get a consensus on which Node versions we are going to support.
+
+In order to provide dependency management consistency, we need a consensus on which npm versions to support.
+
+Staying up to date with the major and minor versions of Node is important:
+
+- to be able to use new features (nice to have)
+- to keep working on a version that receives bug fixes and security updates (must have)            
+
+## Decision
+
+To use 8.x, the current Long Term Support (LTS) line.
```

Comment:
@elifesciences/article-viewer Will this be compatible with eLife's Lens?

Reply:
It looks like I may have used v6.10.3 the last time I built Lens. I will test 8.x and see how it goes.

Reply:
I successfully built Lens using node v8.9.4 today.

Reply:
We have your approval then?

Reply:
Taking the successful build as approval. @gnott shout up if there's anything that argues against this and we can review.